Nuqleon/Core/BCL/Tests.Nuqleon.Memory/System/TupletTests.Generated.cs

```diff
@@ -234,8 +234,8 @@ public void Tuplet3()
             Assert.IsFalse(args1 == args2);
             Assert.IsTrue(args1 != args2);
 
-            Assert.AreNotEqual(args1, "foo");
-            Assert.AreNotEqual(args1, null);
+            Assert.AreNotEqual<object>(args1, "foo");
```

Comment:
Is this intended to be an Equals or a ReferenceEquals compare?

Reply:
My goal was not to change the behaviour. To the best of my knowledge, it was doing an `Equals` style comparison before.

Before it was using the non-generic `AreNotEqual` overload that takes two arguments of type `object`. When they added the generic one it caused those lines of code to become ambiguous.